### ISSUE TITLE
[Bug]: This PDF file contains JavaScript. If you want to view it, please download and open it in your local PDF viewer. 

### ISSUE THREAD

**Issue by abernardgalilee** [pimcore/pimcore#16955]
### Pimcore version

11.2.2

### Steps to reproduce

Pdf with Javascript : 
[PDFS_CopyPastListEntries.pdf](https://github.com/pimcore/pimcore/files/15013260/PDFS_CopyPastListEntries.pdf)
PrintReady Pdf without Javascript can be downloaded here : http://www.folionet.fr/galilee/Arnaud/PERM_VOYAGES_PLV_GAMME_AFRICAN_COLORS_V1.pdf

### Actual Behavior

The PrintReadyPdf file has clearly no Javascript at all and a pro software like Pitstop does not find any malicious javascript in the PdfFile.
I think the streming of the PDF is not good enough.
Javascript can occur in a Pdf when you have forms such as : 
<img width="619" alt="Capture d’écran 2024-04-17 à 17 44 23" src="https://github.com/pimcore/pimcore/assets/13363529/b95bb516-5e2c-4c34-9b18-da252b020a11">

### Expected Behavior

The PrintReadyPdf should not show the warning while the other Pdf should.

**Comment by kingjia90** [comment]
I guess then there is a false positive due to this 
![image](https://github.com/pimcore/pimcore/assets/6014195/6a90a200-3e8f-43c1-a993-01ebd1a2eb47)
https://github.com/pimcore/pimcore/blob/c1d4c2aa198565b645125b3c85adb04fa13ae677/models/Asset/Document.php#L180

**Comment by abernardgalilee** [comment]
Yes sure you are right.
But the algorithm is simply wrong. This is not what should be done in order to detect Javascript in PDFs....

**Comment by kingjia90** [comment]
Yes, i believe there were no any other better solution for that at the time, in the beginning it was even intended to be "sanitizing" instead of just scanning https://github.com/pimcore/pimcore/issues/14998#issuecomment-1742720179 
Maybe there are some better tools/idea now

It would probably make more sense to have it like a soft warning (like old school outlook) when the `/JS` is detected and not completely block it, in addition should have a button "Looks safe" to proceed the preview as usual, as the uploaded files should be trusted source or sanitized at origin.
![image](https://github.com/pimcore/pimcore/assets/6014195/457a4d5c-bd9d-4a09-a5b3-b22adbe90e1a)
